kbignum/src/commonMain/kotlin/com/soywiz/kbignum/BigNum.kt

```diff
@@ -31,6 +31,7 @@ class BigNum(val int: BigInt, val scale: Int): Number() {
         else -> BigNum(int / (10.bi pow (this.scale - otherScale)), otherScale)
     }
 
+    operator fun unaryMinus(): BigNum = this * BigNum("-1")
```

Comment:
What about `BigNum(-int, scale)`? That should be faster I guess

https://github.com/korlibs/kbignum/blob/cf76731478c8fbc95dc6e7c3fed6b19a4d4963ca/kbignum/src/commonMain/kotlin/com/soywiz/kbignum/BigInt.kt#L318

Reply:
Can you add a test here verifying the new `unaryMinus`?

https://github.com/korlibs/kbignum/blob/52cd24dab727fb56c29039e3533a15d70025898e/kbignum/src/commonTest/kotlin/com/soywiz/kbignum/BigNumTest.kt#L7

Reply:
Sure, both suggestions applied.